### ISSUE TITLE
Publish a new release

### ISSUE THREAD

**Issue by Morganamilo** [paritytech/merkle-mountain-range#3]
`sp-mmr-primitives (substrate/primitives/merkle-mountain-range)` can't be released because it has a git dep on `ckb-merkle-mountain-range (https://github.com/paritytech/merkle-mountain-range)` and the public API has changed since the last release `0.6.0 (10 months ago)`

Not releasing `sp-mmr-primitives` would block the release of the following reverse dependencies:

```
mmr-gadget (substrate/client/merkle-mountain-range)
mmr-rpc (substrate/client/merkle-mountain-range/rpc)
pallet-mmr (substrate/frame/merkle-mountain-range)
polkadot-service (polkadot/node/service)
rococo-runtime (polkadot/runtime/rococo)
sp-consensus-beefy (substrate/primitives/consensus/beefy)
westend-runtime (polkadot/runtime/westend)
```

**Comment by Morganamilo** [comment]
Just realised this is a fork of the crates.io crate. That complicates things slightly.

**Comment by Morganamilo** [comment]
I've released `0.7.0` from commit 537f0e3f67c5adf7afff0800bbb81f02f17570a1 with #4 ontop.
